assets/css/_page/_single.scss

```diff
@@ -8,12 +8,18 @@
     line-height: 140%;
   }
 
+.button {
+  margin: 30px auto;
+  display: table;
+  }
```

Comment:
Have you checked how does it look on mobile devices or at least in a narrow browser window? Can you attach a screenshot?

If you are in the same network you can type your computer IP instead of localhost and you should be able to check look & feel on mobiles :)

Reply:
Well, I have tested in on Chrome "mobile mode". It is accessible in Inspect->Toggle device toolbar. It was fine to me.

![buttons](https://user-images.githubusercontent.com/30127736/81477102-27a0b200-9216-11ea-8262-cf910541d2d2.png)